src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -35,19 +27,28 @@
   </ItemGroup>
   
   <ItemGroup>
-    <EmbeddedResource Include="../../bin/DefaultTemplates.tar.gz">
-      <Link>DefaultTemplates.tar.gz</Link>
+    <EmbeddedResource Include="../../bin/Hl7v2DefaultTemplates.tar.gz">
+      <Link>Hl7v2DefaultTemplates.tar.gz</Link>
     </EmbeddedResource>
   </ItemGroup>
   
   <ItemGroup>
     <None Update="oras-linux">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="oras.exe">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
   </ItemGroup>
 
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
-    <Exec Command="mkdir ..\..\bin &amp; tar -zcvf ../../bin/DefaultTemplates.tar.gz -C ../../data/Templates/Hl7v2 ." />
+    <Exec Command="mkdir ..\..\bin &amp; tar -zcvf ../../bin/Hl7v2DefaultTemplates.tar.gz -C ../../data/Templates/Hl7v2 ." />
+  </Target>
+  
+  <Target Name="DownloadOrasFile" BeforeTargets="Build">
+    <DownloadFile SourceUrl="$(OrasWinUrl)" DestinationFolder="../../bin">
+      <Output TaskParameter="DownloadedFile" ItemName="Content" />
+    </DownloadFile>
+    <Exec Command="tar -xvf ..\..\bin\oras_0.8.1_windows_amd64.tar.gz -C ..\..\bin &amp; copy ..\..\bin\oras.exe ." />
```

Comment:
Just want to double confirm that do we always use the windows one? If so, can the tool be run on Linux platform?